app/src/main/java/edu/uc/groupproject/covid19tracker/service/NewsDataProvider.kt

```diff
@@ -1,30 +1,35 @@
 package edu.uc.groupproject.covid19tracker.service
 
+import android.util.Log
 import okhttp3.*
 import org.json.JSONObject
 import java.io.IOException
 
 class NewsDataProvider {
 
     fun getNewsData(dataType: String): ArrayList<String>?  {
-         var client = OkHttpClient()
-         var request:Request = Request.Builder()
+         val client = OkHttpClient()
+         val request:Request = Request.Builder()
```

Comment:
Declaring variables: https://kotlinlang.org/docs/tutorials/kotlin-for-py/declaring-variables.html